spec/spec_helper.rb

```diff
@@ -1,14 +1,13 @@
 require 'asciidoctor-pdf'
+require 'chunky_png'
+require 'fileutils'
 require 'pathname'
 require 'pdf/inspector'
 
 RSpec.configure do |config|
   config.before :suite do
-    FileUtils.mkdir_p output_dir
```

Comment:
Please leave the before and after as they were. The output folder should get cleaned up.

Reply:
I notice this section is now gone. Intention was to have the visual output in spec/output/integration/... in case of differences.